src/components/Loading/Loading.tsx

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect } from 'react'
+import styles from './Loading.module.scss';
+import logo from '../../assets/Temp_Roadwatch_logo.svg';
+// import logo2 from '../../assets/Temp_Roadwatch_logo-2.svg'
+
+export function Loading(): JSX.Element {
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
So, I don't think we need this. This seems to be acting like a set loading time to me, correct me if I'm wrong.
But we want the parent component to request something and then if that takes time then it'll render this. 
Essentially the parent has an isLoading flag not the loading component itself. So I think you can just remove all the JS here and just have a div with the logo centered and its good.

Reply:
Yeah, I was simulating a loading time for 4 seconds, but certainly I will just have the loading be a div with a logo centered in the middle.